app/steps/ui/deceased/anydeceasedchildren/index.js

```diff
@@ -1,12 +1,31 @@
 'use strict';
 
 const ValidationStep = require('app/core/steps/ValidationStep');
+const FormatName = require('app/utils/FormatName');
+const content = require('app/resources/en/translation/deceased/anydeceasedchildren');
+const {get} = require('lodash');
 
 class AnyDeceasedChildren extends ValidationStep {
 
     static getUrl() {
         return '/any-deceased-children';
     }
+
+    getContextData(req) {
+        const ctx = super.getContextData(req);
+        const formdata = req.session.form;
+        ctx.deceasedName = FormatName.format(formdata.deceased);
+        ctx.deceasedDoD = get(formdata, 'deceased.dod_formattedDate');
```

Comment:
Can you clear deceasedName from the ctx by using the action(), also i would have thought that a field would have been created for deceased.dod_formattedData by default, do we need to create it as deceasedDod as well, if we do, it should be removed in the action() as well.

Reply:
Added both to action.
I need the DeceasedDoD in the ctx so that then a field is generated to display on the template.

Reply:
Is there not a field called fields.deceased.dod_formattedDate already that you could have used?